src/components/user/LoginForm.js

```diff
@@ -23,8 +23,8 @@ const messages = defineMessages({
 
 export const LoginForm = ({ handleSubmit, error, submitting, intl }) => (
   <div className="loginForm">
-    <div className="aboutContact">
-      <img src={close} alt="close" className="cross" />
+    <div className="topLeft">
```

Comment:
In CSS you normally write the class names with hyphens instead of camel case. In this case: `top-left`.